awx/main/access.py

```diff
@@ -337,7 +337,10 @@ def get_user_capabilities(self, obj, method_list=[], parent_obj=None, capabiliti
 
             # Actions not possible for reason unrelated to RBAC
             # Cannot copy with validation errors, or update a manual group/project
-            if display_method == 'copy' and isinstance(obj, JobTemplate):
+            if 'write' not in getattr(self.user, 'oauth_scopes', ['write']):
```

Comment:
Is `self.user.oauth_scopes` an `AttributeError` _only_ on basic auth?

Reply:
What do you mean by "only" on basic auth? If it's token auth, then it's set, so maybe you're thinking of some nuances that I am not aware of.

Reply:
Sorry, I was unclear.  I'm trying to avoid as scenario where `self.user.oauth_scopes` is an `AttributeError`, and we unintentionally block access.

Reply:
When using an OAuth 2 token, oauth_scopes should always be set.  If it isn't already the case, I will enforce this with a validator.